front/app/services/permissions/rules/routePermissions.ts

```diff
@@ -66,6 +66,10 @@ export const canAccessRoute = (
       return true;
     }
 
+    if (item.path.includes('folder')) {
```

Comment:
Perhaps a little more explicit, reduce the chance for errors? `.includes('/folders')`? And shall we also check if the user is a project folder moderator (general, not tied to a folder)? Also hard-coded here for the moment.

Reply:
How's this:

``` 
 if (
      item.path.includes('folders') &&
      user &&
      userHasRole(user, 'project_folder_moderator')
    ) {
      return true;
    }
```

Reply:
Fixed this. Will test manually once more since a few changes have been made since I last tested

Reply:
Doing the same!

Reply:
Seems to work on my end as of the latest commit